### PR TITLE
Added `Money` type and `clean-protobuf` command

### LINKED ISSUE #9: How about using type money.proto?
using double in curenncy can be  risky in business, so use money type. (Contains type, int64 unit, int32 subunit{nano})

https://github.com/googleapis/googleapis/blob/master/google/type/money.proto

### PR DESCRIPTION
**Pull request by NikhilSharma03**
## Description

- This PR resolves #9 

- [x] Added `Money` type in place of double
- [x] Added `clean-protobuf` command in Makefile

### CODE REVIEW

**Comment by NikhilSharma03** [comment]
@jinwoo1225 Can you please review it?

**Comment by jinwoo1225** [comment]
sure!

File: server/api/protobuf/okane.proto [hunk 1]
```diff
@@ -4,13 +4,15 @@ package okanepb;
 
 option go_package = "/okanepb";
```
Comment (jinwoo1225) [inline server/api/protobuf/okane.proto]: how about using go_package option as full path of this repository?

```suggestion
option go_package = "github.com/NikhilSharma03/Okane/server/api/protobuf/okane;okanepb";
```
```

**Review by jinwoo1225 — APPROVED** [review]
LGTM! 👍 

just left a comment about option style of an option

File: server/api/protobuf/okane.proto (hunk 1 above)
Comment (NikhilSharma03) [reply server/api/protobuf/okane.proto]: I think that would be better. Updated ✅

**Pull request merged**